content/docs/alerting/configuration.md

```diff
@@ -442,6 +442,15 @@ links:
 
 # The HTTP client's configuration.
 [ http_config: <http_config> | default = global.http_config ]
+
+# The part or component of the affected system that is broken.
+[ component: <string> ]
```

Comment:
These look like tmpl_string to me

Reply:
@brian-brazil I've updated them now to be `tmpl_string`. I don't fully understand the difference, or how to determine when it should be `string` or `tmpl_string`, so any information or links you can provide on that would be appreciated!

Reply:
You need to dig through to where they're used, and see if they're templated - which all 3 of these are.